CONTRIBUTING.md

```diff
@@ -4,33 +4,35 @@ Thanks for your interest in this project.
 
 ## Project description
 
-Jakarta Data provides a specification document, API and TCK for allowing applications and other Jakarta EE components to explore the benefits of easy access to data technologies such as relational and non-relational databases, cloud-based data services, and so on.
+Jakarta Data provides an API to allow easy data access technologies. Thus, a
+Java developer can split the persistence and the model with several features
+such as a Repository interface with the method by query, where the framework
+will implement it.   References:
```

Comment:
Suggested rewording to address a couple of minor grammar issues:

"Jakarta Data provides an API that makes data access easy. A
Java developer can split the persistence from the model with several features,
such as the ability to compose custom query methods on a Repository interface where the framework
will implement it."

The same applies to Readme.md

Reply:
Thanks